packages/imitation_game/tests/smiley/uikit/run_ios.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# TODO: What about customizing the DEVELOPMENT_TEAM?
+set -e
+cd $( dirname "${BASH_SOURCE[0]}" )
+cd smiley
+xcodebuild -scheme smiley -configuration Release -project smiley.xcodeproj -archivePath ./smiley.xcarchive archive
```

Comment:
can't you use flutter tool for this? Maybe add a comment for the reasoning

Reply:
Nope, we didn't make it that generic to be used to launch an arbitrary app.